### PR TITLE
Get various accumulated fixes back into `main`

### PR DESCRIPTION
**Pull request by BGluth**
All of the fixes over the past month have been on `assorted_fixes`. Since we're looking at getting withdrawal support done, we probably start thinking about getting this merged in.

### CODE REVIEW

File: protocol_decoder/src/decoding.rs [hunk 1]
```diff
@@ -227,6 +217,9 @@ impl ProcessedBlockTrace {
             }
         }
 
+        // trie_state.state.insert(Nibbles::from_h256_be(hash(H160::zero().as_bytes())),
+        // ValOrHash::Val(EMPTY_ACCOUNT_BYTES_RLPED.to_vec()));
+
```
Comment (Nashtare) [inline protocol_decoder/src/decoding.rs]: nit: can be removed?

File: protocol_decoder/src/utils.rs [hunk 2]
```diff
@@ -44,3 +45,15 @@ fn print_value_and_hash_nodes_of_trie_common(trie: &HashedPartialTrie) -> Vec<St
         })
         .collect()
 }
+
+pub(crate) fn h_addr_nibs_to_h256(h_addr_nibs: &Nibbles) -> H256 {
+    // TODO: HACK! This fix really needs to be in `eth_trie_utils`...
```
Comment (Nashtare) [inline protocol_decoder/src/utils.rs:50]: agreed 😝

**Review by Nashtare — APPROVED** [review]
LGTM, thanks!

File: protocol_decoder/src/decoding.rs (hunk 1 above)
Comment (BGluth) [reply protocol_decoder/src/decoding.rs]: Missed that, thanks!

**Comment by Nashtare** [comment]
@BGluth Now that `assorted-fixes` is merged to `main`, should we bump the deps on `zero-bin:main` as well?

**Comment by BGluth** [comment]
Yeah I'll open a PR for that pretty soon. :+1:

**Comment by Nashtare** [comment]
nit: it's already included in my PR for the `VerifierState`, so probably not needed 🙂

**Pull request merged**